### ISSUE TITLE
Internal links should respect chosen version

### ISSUE THREAD

**Issue by Hyperseeker** [nornagon/cdda-guide#162]
**TL;DR**: if a specific version had been chosen (via URL or the `<select>` menu at the bottom), links to internal entries should be appended with that version. If `?v=0.G`, then every link to another item/monster/flag/proficiency/etc. should also have `?v=0.G` at the end.

I mod for 0.G exclusively, and most of the content I browse on HHG is for that version. In fact, my browser has saved the link to HHG with `?v=0.G` at the end. I'd like HHG to respect that choice whenever I navigate the app.

The reason I ask for the version tag to be appended to links is because I frequently open multiple links from the same search (e.g., `makeshift` was the most recent) in new tabs. It could be 2 or 20 tabs, depending on the context. Going through each tab and manually setting version gets tiresome very quickly.

If I've already chosen a version tag, it's a pretty clear indicator that I want to browse the catalog at that version. If I need the data from the latest version, I just manually select it (or use a URL without a version tag).

(Usually I also add a solution, what I would've implemented, to be respectful of the developer's time. Unfortunately, I couldn't make heads or tails of the Svelte app, and I don't have enough time to learn the framework right now.)

**Comment by nornagon** [comment]
Yep, agree. I’ll fix this! Thanks for the report.
